services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+
+from django.db.models.query import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime.datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:

```

Comment:
Indicate a `QuerySet` of what instances you expect to return in the the return type annotation here and in the other functions